predicate-push-down.md

```diff
@@ -3,4 +3,142 @@ title: 谓词下推
 category: performance
 ---
 
-# 谓词下推
\ No newline at end of file
+# 谓词下推
+
+谓词下推将查询语句中的过滤表达式计算尽可能下推到距离数据源最近的地方，以尽早完成数据的过滤，进而显著地减少数据传输或计算的开销。
+
+## 示例
+
+以下通过一些例子对谓词下推优化进行说明，其中示例1/2/3为谓词下推适用的案例，示例4/5/6为谓词下推不适用的案例。
+
+### 示例 1
```

Comment:
Can we use a more specific heading than this? Like, specify which type of predicates.